docs-site/docs/autograd.md

```diff
@@ -148,3 +148,863 @@ for epoch in range(epochs):
         for p, gp in autograd.backward(loss):  # auto backward
             sgd.update(p, gp)
 ```
+
+### Supported Operations
```

Comment:
if the descriptions of the operations are already included in the docString, then there is no need to list them here.
Otherwise, we have to maintain both the docString and this file when there is a change to any API.

Reply:
operation list here is different from autograd, we do not support all operators in the autograd. 
By the way, I reformat it now, I guess maybe more clear now.

Reply:
and, the name of operators may be different between ONNX and SINGA, the list here is ONNX operators' name.

Reply:
If these operations are onnx operators, then we just need to list these operators in onnx.md and refer to the onnx website for the descriptions.
This is autograd.md, which should only include descriptions about autograd.

Reply:
Sorry, I misunderstood what you said. I will delete this part and add a link to the API reference pages.